app/react/App/scss/modules/_document.scss

```diff
@@ -31,7 +31,15 @@
 }
 
 .document-viewer .main-wrapper {
-  pre.force-rtl {
+  .raw-text {
+    white-space: pre;
+
+    @media (max-width: 1023px) {
+      white-space: pre-line;
+    }
+  }
```

Comment:
I know that Uwazi has a mix of max-width and min-width media queries.  Still, for new developments, lets try to do it "Mobile first".  So, the "normal" behavior is pre-line and at min-width 1024px it starts using pre.